ui/app/templates/vault/cluster/policies/index.hbs

```diff
@@ -69,21 +69,16 @@
         </div>
       {{else}}
         <LinkedBlock
-          @params={{array "vault.cluster.policy.show" this.policyType item.id}}
+          @params={{array "vault.cluster.policies.policy.show" this.policyType item.id}}
           class="list-item-row"
           data-test-policy-link={{item.id}}
         >
-          <div class="columns is-mobile">
+          <div class="columns is-mobile" data-test-policy-item>
             <div class="column is-10">
-              <LinkTo
-                @route="vault.cluster.policy.show"
-                @models={{array this.policyType item.id}}
-                class="has-text-black has-text-weight-semibold"
-                data-test-policy-item={{true}}
-              >
-                <Icon @name="file" class="has-text-grey-light" />
-                <span class="is-underline" data-test-policy-name>{{item.id}}</span>
-              </LinkTo>
+              <Icon @name="file" class="has-text-grey-light" />
```

Comment:
The `<LinkTo>` inside the `<LinkedBlock>` felt redundant, but I can put back if there's a reason we do this!